src/fides/connectors/aws.py

```diff
@@ -295,6 +297,7 @@ def create_rds_systems(
             fides_key=instance["DBInstanceIdentifier"],
             name=instance["DBInstanceIdentifier"],
             description=f"Fides Generated Description for RDS Instance: {instance['DBInstanceIdentifier']}",
+            meta={(pair["Key"], pair["Value"]) for pair in instance["TagList"]},
```

Comment:
Similar here, account and handle the scenario where `TagList` doesn't exist ([checks](https://github.com/ethyca/fides/actions/runs/9914563221/job/27394125529?pr=4998#step:7:918))